src/plays/registration-form/RegistrationForm.jsx

```diff
@@ -0,0 +1,238 @@
+import { getPlayById } from "meta/play-meta-util";
+
+import PlayHeader from "common/playlists/PlayHeader";
+import "./Registration-form.css";
+import { useState } from "react";
+
+function RegistrationForm(props) {
+  // Do not remove the below lines.
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+
+  //for input values
+  const [values, setValues] = useState({
+    name: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+    errors: {
+      name: "",
+      email: "",
+      password: "",
+      confirmPassword: "",
+    },
+  });
+
+  //input values after submit
+  const [storedValues, setStoredValues] = useState({
```

Comment:
Why do you need to maintain two states? Why can't the value states itself can be leveraged when the form submits?